src/components/Payment/BraintreeDropInInterface.vue

```diff
@@ -498,3 +414,79 @@ $border-width: 1px;
 	text-align: center;
 }
 </style>
+<style lang="postcss">
+	/* New Tailwind Styles */
+
+	/* Braintree section headings
+	'Choose a way to pay'
+	'Paying with Card'
+	'Other ways to pay' */
+	#dropin-container [data-braintree-id="choose-a-way-to-pay"],
+	#dropin-container [data-braintree-id="methods-label"],
+	#dropin-container [data-braintree-id="other-ways-to-pay"] {
+		@apply tw-text-h4 tw-text-primary tw-text-left;
+	}
+
+	/* Payment method form headers text
+	'Pay with card'
+	'Paypal'
+	'GooglePay'
+	'ApplePay' */
+	#dropin-container [data-braintree-id="paypal-sheet-header"] .braintree-sheet__text,
+	#dropin-container [data-braintree-id="apple-pay-sheet-header"] .braintree-sheet__text,
+	#dropin-container [data-braintree-id="google-pay-sheet-header"] .braintree-sheet__text,
+	#dropin-container [data-braintree-id="card-sheet-header"] .braintree-sheet__text,
+	#dropin-container [data-braintree-id="paypal-sheet-header"] .braintree-sheet__label,
+	#dropin-container [data-braintree-id="apple-pay-sheet-header"] .braintree-sheet__label,
+	#dropin-container [data-braintree-id="google-pay-sheet-header"] .braintree-sheet__label,
+	#dropin-container [data-braintree-id="card-sheet-header"] .braintree-sheet__label {
+		@apply tw-ml-0 tw-text-h4 tw-text-primary tw-text-left;
+	}
+
+	/* Saved payment methods container
+	List of vaulted cards */
+	#dropin-container [data-braintree-id="methods-container"] .braintree-method {
+		@apply tw-border-solid tw-border-tertiary tw-w-full tw-border;
+	}
+
+	#dropin-container [data-braintree-id="methods-container"] .braintree-method:first-child {
+		@apply tw-rounded-tr-sm tw-rounded-tl-sm tw-rounded-bl-none tw-rounded-br-none;
+	}
+
+	#dropin-container [data-braintree-id="methods-container"] .braintree-method:last-child {
+		@apply tw-rounded-br-sm tw-rounded-bl-sm;
+	}
+
+	/* Payment Option buttons on initial UI */
+	#dropin-container [data-braintree-id="payment-options-container"] .braintree-option {
+		@apply tw-border-solid tw-border-tertiary tw-w-full tw-border tw-border-b-0;
+	}
+
+	#dropin-container [data-braintree-id="payment-options-container"] .braintree-option:first-child {
+		@apply tw-rounded-tr-sm tw-rounded-tl-sm tw-rounded-bl-none tw-rounded-br-none;
+	}
+
+	#dropin-container [data-braintree-id="payment-options-container"] .braintree-option:last-child {
+		@apply tw-rounded-br-sm tw-rounded-bl-sm tw-border-b;
+	}
+
+	/* 'Choose another way to pay' text */
+	#dropin-container [data-braintree-id="toggle"],
+	#dropin-container [data-braintree-id="toggle"]:hover {
+		@apply tw-bg-transparent;
+	}
+
+	#dropin-container [data-braintree-id="toggle"] span,
+	#dropin-container [data-braintree-id="toggle"] span:focus,
+	#dropin-container [data-braintree-id="toggle"] span:hover {
+		@apply tw-text-base tw-no-underline tw-border-0;
+		@apply tw-font-medium;
```

Comment:
This actually needs to be on a second line to guarantee the order, otherwise tw-text-base font-weight overwrites tw-font-medium font weight